README.md

```diff
@@ -1105,6 +1110,59 @@ However, the following rules of thumb are useful to keep in mind:
   Alternatively, the entire OrientDB class and all edges that may point to it may be omitted
   entirely from the GraphQL schema.
 
+## Query cost estimation
+
+Due to GraphQL's easy-to-use nature, queries that use a lot of memory and/or have a long execution
+time can happen often. For better user-friendliness, it's worthwhile to estimate a query's cost (in
```

Comment:
Nit: "For better user-friendliness," -> "To prevent queries from using too many resources" (too much memory and/or time? or too many resources?)

Reply:
Removed this.